Source/CsDebugScript.Engine/Engine/Utility/SimpleCache.cs

```diff
@@ -91,6 +94,14 @@ public T Value
                 Cached = true;
             }
         }
+
+        /// <summary>
+        /// Invalidate cache entry.
+        /// </summary>
+        public void InvalidateCache()
+        {
+            Cached = false;
+        }
```

Comment:
Not sure that this is necessary. Please see how this is internally used already... Search for references of Cached property...